### PR TITLE
Retry UEFI boot Configuration

### PR DESCRIPTION
**Pull request by derekhiggins**
This consistently fails on some HW, retry until we
find a proper solution.

patch 1 matches what was merged downstream and patch 2 I've added to improve on the code.

### CODE REVIEW

File: hardware_manager/ironic_coreos_install.py [hunk 1]
```diff
@@ -104,8 +104,17 @@ def install_coreos(self, node, ports):
         boot = hardware.dispatch_to_managers('get_boot_info')
         if boot.current_boot_mode == 'uefi':
             LOG.info('Configuring UEFI boot from device %s', root)
-            efi_utils.manage_uefi(root)
-
+            for i in range(3):
```
Comment (dtantsur) [inline hardware_manager/ironic_coreos_install.py:107]: nit: I'd use a constant for readability

**Comment by derekhiggins** [comment]
Posted downstream here https://github.com/openshift/ironic-agent-image/pull/45

**Pull request merged**